src/AspNetCoreMulti/Example/GraphQL/Operations.cs

```diff
@@ -0,0 +1,69 @@
+using Example.Repositories;
+using GraphQL.Types;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+
+namespace Example.GraphQL
+{
+    public interface IOperation
+    {
+        IEnumerable<IFieldType> RegisterFields();
+    }
+
+    public interface IDogOperation : IOperation { }
+
+    public interface ICatOperation : IOperation { }
+
+    public class DogOperation : ObjectGraphType<object>, IDogOperation
+    {
+        private readonly IServiceProvider _serviceProvider;
+
+        public DogOperation(IServiceProvider serviceProvider)
+        {
+            _serviceProvider = serviceProvider;
+        }
+
+        public IEnumerable<IFieldType> RegisterFields()
+        {
+            var fields = new List<IFieldType>
+            {
+                Field<StringGraphType>("say", resolve: context => "woof woof woof"),
```

Comment:
`Field<StringGraphType>(` **appends** field to graph type and returns reference to it.